### PR TITLE
Allow to run test suite from release tarball

### LINKED ISSUE #4430: Allow to execute the test suite from the content of release tarball.
I'd appreciate, if the content of release tarball was complete, so the included test suite can be executed. Currently, trying to execute the test suite from the tarball, it fails:

~~~
$ rake
rake aborted!
LoadError: cannot load such file -- util/automatiek.rake
~~~

Alternatively, please make sure the test suite is executable via command as simple as:

~~~
$ ruby -e 'Dir.glob "./test/**/test_*.rb", &method(:require)'
~~~

Then the tarball could stay in its current from and not all content required by Rake would be necessary.

### PR DESCRIPTION
**Pull request by voxik**
Just skip the Automatie if not avialable. It is not essential to run the test suite.

Fixes #4430

### CODE REVIEW

**Comment by voxik** [comment]
Now I found that it used to be guarded [this](https://github.com/rubygems/rubygems/commit/939d668bad2737e2b956ed914a3667d2752f2862) way

**Comment by simi** [comment]
@voxik that looks ok to me as well.

**Comment by deivid-rodriguez** [comment]
I think the current version is better. Since this is not backed up by tests, the comment you added will prevent me from removing it :laughing:.

File: Rakefile [hunk 1]
```diff
@@ -58,16 +58,20 @@ RDoc::Task.new :rdoc => 'docs', :clobber_rdoc => 'clobber_docs' do |doc|
...
-# We currently ship Molinillo master branch as of
-# https://github.com/CocoaPods/Molinillo/commit/7cc27a355e861bdf593e2cde7bf1bca3daae4303
-Automatiek::RakeTask.new("molinillo") do |lib|
-  lib.version = "master"
-  lib.download = { :github => "https://github.com/CocoaPods/Molinillo" }
-  lib.namespace = "Molinillo"
-  lib.prefix = "Gem::Resolver"
-  lib.vendor_lib = "lib/rubygems/resolver/molinillo"
+begin
+  load "util/automatiek.rake"
```
[the remark this answers is not in the document]
Comment (deivid-rodriguez) [reply Rakefile:62]: Actually I like this suggestion. Checking for presence of a relative `util/automatiek.rake` file before loading it makes things cleaner :+1:.

[the remark this answers is not in the document]
Comment (voxik) [reply Rakefile:64]: Updated

**Review by deivid-rodriguez — APPROVED** [review]

**Comment by deivid-rodriguez** [comment]
Thank you!

**Pull request merged**